crates/sncast/tests/e2e/declare.rs

```diff
@@ -82,8 +85,18 @@ async fn test_happy_case(class_hash: Felt, account_type: AccountType) {
         URL,
         "--contract-name",
         "Map",
-        "--max-fee",
-        "99999999999999999",
+        "--l1-gas",
+        "100000",
+        "--l1-gas-price",
+        "10000000000000",
+        "--l2-gas",
+        "1000000000",
+        "--l2-gas-price",
+        "100000000000000000000",
+        "--l1-data-gas",
+        "100000",
+        "--l1-data-gas-price",
+        "10000000000000",
```

Comment:
note: Using auto estimation fails because fee estimate returns insufficient resources (it's on devnet side). Due to it, we use fixed resource bounds in our tests (applies to all other places).